org.mwc.cmap.plotViewer/src/org/mwc/cmap/plotViewer/actions/ExportWMF.java

```diff
@@ -78,14 +89,49 @@ public static void init(final ToolParent theParent)
 	 */
 	protected void execute()
 	{
+		IRunnableWithProgress runnable = new IRunnableWithProgress()
+		{
+			
+			@Override
+			public void run(final IProgressMonitor monitor)
+					throws InvocationTargetException, InterruptedException
+			{
+				executeInJob(monitor);
+			}
+		};
+		
+		try
+		{
+			ProgressMonitorDialog pmd = new ProgressMonitorDialog(getShell());
+			pmd.run(false, true, runnable);

```

Comment:
Hmm, when I run "Export to WMF", the dialog shows, then Debrief freezes while the export happens.  Since I'm on ubuntu, Debrief goes dark whilst the app is frozen.

Is there a way of doing this so that the App remains "live"?  I guess this means so that the export happens in the background.